hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/bucket/OMBucketCreateRequest.java

```diff
@@ -205,14 +205,17 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
       // Add default acls from volume.
       addDefaultAcls(omBucketInfo, omVolumeArgs);
 
+      // update used namespace for volume
+      omVolumeArgs.incrUsedNamespace(1L);
+
```

Comment:
Before this update, we need to do the namespace quota check in checkQuotaBytesValid.

Reply:
Also updated the test to test quota exceed case.